engine/src/main/java/org/terasology/rendering/nui/layers/mainMenu/JoinGameScreen.java

```diff
@@ -352,6 +357,21 @@ public Boolean get() {
                 refresh();
             });
         }
+
+        UIButton broadCastServerButton = find("broadcast", UIButton.class);
+        if (broadCastServerButton != null) {
+            broadCastServerButton.bindEnabled(new ReadOnlyBinding<Boolean>() {
+                @Override
+                public Boolean get() {
+                    return broadCastServer.isTurnOnBroadcast();
```

Comment:
```suggestion
                    return !broadCastServer.isTurnOnBroadcast();
```
I think you probably would want to enable the button when not broadcasting and disable it when it is.

Reply:
Yeah thanks for the suggestion :+1: